api/src/pcapi/routes/backoffice_v3/collective_offers/blueprint.py

```diff
@@ -318,5 +322,116 @@ def batch_reject_collective_offers() -> utils.BackofficeResponse:
         flash("L'un des identifiants sélectionnés est invalide", "danger")
         return redirect(request.referrer or url_for("backoffice_v3_web.collective_offer.list_collective_offers"), 303)
 
-    _batch_validate_or_reject_collective_offers(OfferValidationStatus.REJECTED, form.object_ids_list)
+    _batch_validate_or_reject_collective_offers(offer_mixin.OfferValidationStatus.REJECTED, form.object_ids_list)
     return redirect(request.referrer or url_for("backoffice_v3_web.collective_offer.list_collective_offers"), 303)
+
+
+def _is_collective_offer_price_editable(collective_offer: educational_models.CollectiveOffer) -> bool:
+    # if the offer has no stock it has no price therefore its price cannot be updated
+    if not collective_offer.collectiveStock:
```

Comment:
Comme tu charges avec `get_or_404()` l'objet que tu passes en paramètre, es-tu sûr que cet accès ne fait pas "bêtement" une requête N+1 ?
Pour l'éviter, peut-être simplement : 
```suggestion
    if not collective_offer.collectiveStockId:
```

Reply:
Certes, mais c'est dommage de faire directement une N+1.
Autant faire un joinedload dès le départ en récupérant l'objet.